x-pack/plugins/security_solution/public/common/lib/telemetry/events/manual_rule_run/index.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TelemetryEvent } from '../../types';
+import { TelemetryEventTypes } from '../../constants';
+
+export const manualRuleRunOpenModalEvent: TelemetryEvent = {
+  eventType: TelemetryEventTypes.ManualRuleRunOpenModal,
+  schema: {
+    type: {
+      type: 'keyword',
+      _meta: {
+        description: 'Open manual rule run modal (single|bulk)',
+        optional: false,
+      },
+    },
+  },
+};
+
+export const manualRuleRunExecuteEvent: TelemetryEvent = {
+  eventType: TelemetryEventTypes.ManualRuleRunExecute,
+  schema: {
+    rangeInMs: {
+      type: 'integer',
+      _meta: {
+        description: 'Execute manual rule run with range in milliseconds',
```

Comment:
Should this be describing `rangeInMs` exclusively, or the full event?

```suggestion
        description: 'The time range (expressed in milliseconds) against which the manual rule run was executed',
```